### PR TITLE
PP-9334: Add entity for Payment Instrument

### PR DESCRIPTION
**Pull request by gidsg**
Co-authored-by: Alan Carter <[alan.carter@digital.cabinet-office.gov.uk](mailto:alan.carter@digital.cabinet-office.gov.uk)>

## WHAT YOU DID
Add entity for Payment Instrument (as added to the DB in https://github.com/alphagov/pay-connector/pull/3525)

## How to test

As this is just the entity not sure if it is usefully tested on its own?

## Code review checklist

Code should match database migration

### Logging

- [ ] only emit log lines at ERROR level which require immediate attention from a support engineer. These will trigger a zendesk alert.

### Documentation

- [ ] Updated [README.md](http://readme.md/) for any of the following ?

* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/paymentinstrument/model/PaymentInstrumentEntity.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    }
+
+    public void setCardDetails(CardDetailsEntity cardDetails) {
+        this.cardDetails = cardDetails;
+    }
+
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    @JsonProperty("status")
+    @JsonSerialize(using = ToLowerCaseStringSerializer.class)
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/paymentinstrument/model/PaymentInstrumentEntity.java:116]: Is lower-case consistent with what we do elsewhere?

Comment (gidsg) [reply src/main/java/uk/gov/pay/connector/paymentinstrument/model/PaymentInstrumentEntity.java:116]: @alexbishop1 Took this style from https://github.com/alphagov/pay-connector/blob/8b971c830fc607443026244ae4e4db31d0b8efbb/src/main/java/uk/gov/pay/connector/charge/model/CardDetailsEntity.java#L57-L61 so it's at least consistent with what we do in _some_ instances!

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/connector/paymentinstrument/model/PaymentInstrumentEntity.java:116]: Good enough for me…

**Review by alexbishop1 — APPROVED** [review]
Agree there’s not anything of substance to test yet (and no risk from not having tests because this isn’t doing anything yet).

**Pull request merged**